packages/config/config/devices/0x0129/templates/yale_template.json

```diff
@@ -331,5 +331,136 @@
 		"readOnly": false,
 		"writeOnly": false,
 		"allowManualEntry": true
+	},
+	"alarm_map_keypad_limit": {
+		"from": {
+			"alarmType": 161,
+			"alarmLevel": 1
+		},
+		"to": {
+			"notificationType": 6,
+			"notificationEvent": 16
```

Comment:
Please also comment those

Reply:
Happy to, but where did you find the names? They aren't in the Yale docs. Did you look them up in the CC or something?

Reply:
https://github.com/zwave-js/node-zwave-js/blob/master/packages/config/config/notifications.json
type is the first layer key, e.g. 0x06 for Access Control and the event labels can be found under `events` for that type, e.g. [here](https://github.com/zwave-js/node-zwave-js/blob/3b5435d3694534bf5ac152eb40912cb9acf3e46f/packages/config/config/notifications.json#L536)